time_tracker_api/time_entries/time_entries_dao.py

```diff
@@ -225,10 +225,14 @@ def create(self, data: dict):
 
     def update(self, id, data: dict, description=None):
         event_ctx = self.create_event_context("update", description)
-
         time_entry = self.repository.find(id, event_ctx)
         self.check_whether_current_user_owns_item(time_entry)
 
+        if data.get('update_last_entry_if_overlap'):
```

Comment:
`data.get('update_last_entry_if_overlap', None)` or `data.get('update_last_entry_if_overlap', False)`

Reply:
get method return None by default without passing default value, but I can add None if you think it is necessary.